src/index.html

```diff
@@ -5,8 +5,404 @@
+        </div>
+
+        <div class="stars">
+          <div class="stars__container stars__container--active-4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="secondary-text">Reviews: 5</p>
+
+        </div>
+
+        <div class="card__price">
+          <p class="card__priceText">Price:</p>
+
+          <p class="card__priceNumber">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          data-qa="card-hover"
+          class="card__button-link card__button-link--hover"
+        >
+              Buy
+        </a>
+      </section>
+
+      <section class="card">
+        <img
+            src="./images/imac.jpeg"
+            alt="The picture of iMac."
+            class="card__img"
+        >
+
+        <div class="card__product">
+          <p class="card__title">APPLE A1419 iMac 27" Retina</p>
+
+          <p class="card__title">5K Monoblock (MNED2UA/A)</p>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="stars">
+          <div class="stars__container stars__container--active-4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="secondary-text">Reviews: 5</p>
+
+        </div>
+
+        <div class="card__price">
+          <p class="card__priceText">Price:</p>
+
+          <p class="card__priceNumber">$2,199</p>
+        </div>
+
+        <a href="#" class="card__button-link card__button-link--hover">
+              Buy
+        </a>
+      </section>
+
+      <section class="card">
+        <img
+            src="./images/imac.jpeg"
+            alt="The picture of iMac."
+            class="card__img"
+        >
+
+        <div class="card__product">
+          <p class="card__title">APPLE A1419 iMac 27" Retina</p>
+
+          <p class="card__title">5K Monoblock (MNED2UA/A)</p>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="stars">
+          <div class="stars__container stars__container--active-4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="secondary-text">Reviews: 5</p>
+
+        </div>
+
+        <div class="card__price">
+          <p class="card__priceText">Price:</p>
+
+          <p class="card__priceNumber">$2,199</p>
+        </div>
+
+        <a href="#" class="card__button-link card__button-link--hover">
+              Buy
+        </a>
```

Comment:
Need to fix it everywhere
```suggestion
        <a href="#" class="card__button">
          Buy
        </a>
```